### ISSUE TITLE
Commands no work

### ISSUE THREAD

**Issue by fedorablewill** [CitizensDev/Citizens#674]
Whenever I type in any npc command (by which i mean /npc and sub-commands including /npc help) it says
"The command was not recognized
type /npc help for a list of commands"
and nothing in the console. I am using Tekkit v1.1.4

**Comment by Axol96** [comment]
Im having the exact same problem!

**Comment by Axol96** [comment]
Maybe it conflicts with another plugin?!? Idk :S , ima try install it on a clean server

**Comment by Axol96** [comment]
Hmmmm , what version of craftbukkit are you useing : i saw i used a 1.1 snapshot , i gonna try replace it with the actual 1.1 craftbukkit

**Comment by Axol96** [comment]
Yeah i found out what problem it was (well worked for me) ..... I used craftbukkit-1.1-r1.snapshot.jar    u have to download the latest recommendet build wich is craftbukkit-1.1-R1.jar  !!! hope it works !" :D

**Comment by fedorablewill** [comment]
I was using Tekkit v1.1.4
